### PR TITLE
Change all default info logs to their respective levels

### PR DESCRIPTION
**Pull request by baymac**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/gitlabbranchsource/helpers/GitLabPipelineStatusNotifier.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ private static void sendNotifications(Run<?, ?> build, TaskListener listener) {
             return;
         }
         Result result = build.getResult();
-        LOGGER.info("Result: " + result);
+        LOGGER.log(Level.FINE, "Result: " + result);
```
Comment (guillaumelecerf) [inline src/main/java/io/jenkins/plugins/gitlabbranchsource/helpers/GitLabPipelineStatusNotifier.java]: no `String.format()` here?

**Comment by casz** [comment]
@baymac 

I found a few more logs that are noisy:

```
jenkins_1  | 2019-12-10 14:12:52.025+0000 [id=26]	INFO	i.j.p.g.GitLabSCMSource#afterSave: Mode of web hook: SYSTEM
jenkins_1  | 2019-12-10 14:12:52.025+0000 [id=26]	INFO	i.j.p.g.GitLabSCMSource#afterSave: Mode of system hook: SYSTEM
jenkins_1  | 2019-12-10 14:12:52.221+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#retrieve: c, o, e, l..Executor #-1 for master : executing BranchIndexing[Shared/CompileComputeShaders]
jenkins_1  | 2019-12-10 14:12:53.008+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: 1.0.2
jenkins_1  | 2019-12-10 14:12:53.009+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.115+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: features/improve-error-logging
jenkins_1  | 2019-12-10 14:12:53.116+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.213+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: master
jenkins_1  | 2019-12-10 14:12:53.214+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.266+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#retrieveActions: h, e, l..Executor #-1 for master : executing BranchIndexing[Shared/CompileComputeShaders]
jenkins_1  | 2019-12-10 14:12:53.320+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: update-fxc
jenkins_1  | 2019-12-10 14:12:53.321+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.406+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: 2.0.0
jenkins_1  | 2019-12-10 14:12:53.407+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.475+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#retrieveActions: h, e, l..Executor #-1 for master : executing BranchIndexing[Shared/CompileComputeShaders]
jenkins_1  | 2019-12-10 14:12:53.545+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: v1.2.3
jenkins_1  | 2019-12-10 14:12:53.545+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.635+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: v1.2.2
jenkins_1  | 2019-12-10 14:12:53.636+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.732+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: v1.2.1
jenkins_1  | 2019-12-10 14:12:53.732+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.854+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: v1.2.0
jenkins_1  | 2019-12-10 14:12:53.854+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:53.938+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource#createProbe: Creating a probe: v1.1.0
jenkins_1  | 2019-12-10 14:12:53.938+0000 [id=6854]	INFO	i.j.p.g.GitLabSCMSource$4#stat: Path of file: Jenkinsfile
jenkins_1  | 2019-12-10 14:12:54.240+0000 [id=6854]	INFO	j.b.MultiBranchProject$BranchIndexing#run: Shared/CompileComputeShaders #20191210.141252 branch indexing action completed: SUCCESS in 2.2 sec
```

**Comment by baymac** [comment]
@casz do you want to remove these? Currently they are at either fine or finest levels.

**Comment by casz** [comment]
Oh I missed them because they were collapsed 😓

**Comment by casz** [comment]
But ya I am fine removing some as they were used as debugging which you can achieve by attaching the debugger :)

**Comment by casz** [comment]
I'd like to release this as the logs seem to slow down our Jenkins instance with several hundred projects over several groups.

**Pull request merged**